nilearn/_utils/docs.py

```diff
@@ -871,16 +871,13 @@ def custom_function(vertices):
 docdict[
     "symmetric_cbar"
 ] = """
-symmetric_cbar : :obj:`bool`, or "auto", optional
-    Specifies whether the colorbar should range from `-vmax` to `vmax`
-    or from `vmin` to `vmax`.
-    Setting to `"auto"` will select the latter
-    if the range of the whole image is either positive or negative.
-
-    .. note::
-
-        The colormap will always range from `-vmax` to `vmax`.
-
+symmetric_cbar : :obj:`bool`, or "auto", default="auto"
+    Specifies whether the colorbar and colormap should range from `-vmax` to
+    `vmax` (or from `vmin` to `-vmin` if `-vmin` is greater than `vmax`) or
+    from `vmin` to `vmax`.
+    Setting to `"auto"` will select the former if either 1) `vmin` or `vmax` is
+    None and the image has both positive and negative values, or 2) `vmin` is
+    equal to `-vmax`.
```

Comment:
If  `vmin`  is   equal to `-vmax`, then the two behaviors are equivalent, no ?

Reply:
Yes, you're right. Updated the docstring